src/lib/accessibility/Behaviors/Button/ToggleButtonBehavior.ts

```diff
@@ -0,0 +1,33 @@
+import { IAccessibilityBehavior, ComponentState } from '../../interfaces'
+
+export class ToggleButtonBehavior implements IAccessibilityBehavior<{}, {}> {
+  private attributes = {
+    'ms-acc-behavior': this.name,
+    role: 'button',
+  }
+
+  public get name(): string {
+    return 'toggle-button'
+  }
+
+  public generateAriaAttributes(props, state): object {
```

Comment:
Let's update this signature to take a single object.  That way if a prop moves to state or vice versa, it doesn't break the behavior.  It also makes the behaviors not dependent on React patterns.